.gitignore

```diff
@@ -114,3 +114,6 @@ cms/fragments*
 !portality/static/doaj/css/*_widget.css
 
 portality/static/widget/fq_widget_build_info.txt
+
+#json for harvester test
+doajtest/unit/resources/harvester_resp.json
```

Comment:
Should this file be in the gitignore?  It's also in the PR

Reply:
Well, this file is changed every single time the harvester test is run and I don't think we want to have it in every single PR later. That is why I added that to the repo and then set github to ignore it. Not sure what is the best solution here though.